sdk-tests/src/test/java/io/dapr/it/actors/app/TestApplication.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Dapr's HTTP callback implementation via SpringBoot.
  */
-@SpringBootApplication(scanBasePackages = {"io.dapr.springboot", "io.dapr.it.actors.app"})
```

Comment:
Why is `"io.dapr.it.actors.app"` no longer needed?

Reply:
same answer as below.